.github/bin/validate-package.mjs

```diff
@@ -0,0 +1,456 @@
+      target.endsWith('/') ||
+      target.endsWith('.d.ts')
+    ) {
+      continue;
+    }
+
+    // `import.meta.resolve` is used here to determine
+    // where the import would be resolved from, following
+    // any symlinks. Since this package is linked, it
+    // should never have `node_modules` in the resolved
+    // path. It *could* happen if a dev is writing code
+    // within a parent folder named as such, but that is
+    // unsafe anyways.
+    // -------------------------------------------------
+    // If this is ever setup to run in the post-deploy
+    // test, then this will cause issues as that runs
+    // from this folder specifically.
+    if (import.meta.resolve(target).includes('node_modules')) {
+      exitCode++;
+      summary += `| \`${target}\` | ✗ Resolves to node_modules |\n`;
+      console.error(`✗ ${target} resolves to node_modules`);
+      continue;
+    }
+
+    try {
+      let version = '';
+      if (target.endsWith('.json')) {
+        const data = await import(target, { with: { type: 'json' } });
+        version = Object.keys(data.default).at(-1);
+      } else {
+        const axe = await import(target);
+
+        if (!axe.default?.version) {
+          throw new Error('Missing version property');
+        }
+
+        version = axe.default.version;
+        if (version !== pkg.version) {
+          console.error(
+            `✗ ${target} version mismatch: expected ${pkg.version}, got ${version}`
+          );
+          summary += `| \`${target}\` | ✗ Version Mismatch | ${version} |\n`;
+          anyCaught = true;
+          continue;
+        }
+      }
+      console.info(`✓ ${target}`);
+      summary += `| \`${target}\` | ✓ Importable | ${version} |\n`;
+    } catch (error) {
+      console.error(`✗ ${target}`);
+      summary += `| \`${target}\` | ✗ Not Importable |\n`;
+      summary += `\n\`\`\`\n${error.message}\n\`\`\`\n`;
+      anyCaught = true;
+    }
+  }
+
+  if (anyCaught) {
+    exitCode++;
+  }
+
+  await appendToSummaryFile(summary);
+};
+
+/**
+ * When a PR targets `master` or a `release-*` branch,
+ * or these branches are pushed to, we run SRI validation.
+ * Otherwise, it is skipped since the SRI hashes are only
+ * updated when releasing.
+ *
+ * The history file is deprecated. However, until it is removed
+ * we should be prudent and continue to validate it.
+ */
+const validateSriHashes = async () => {
+  const currentBranch =
+    process.env.GITHUB_REF_NAME || process.env.GITHUB_HEAD_REF || '';
+
+  if (!/^release-.+/.test(currentBranch) && currentBranch !== 'master') {
+    console.log(`Skipping SRI validation (current branch: ${currentBranch})`);
+    return;
+  }
+
+  summary += `\n## Subresource Integrity Check
+
+This check validates the current build against the SRI hash
+for the version defined in \`sri-history.json\`.
+
+| File | Status |
+|------|--------|
+`;
+
+  const sriHistory = await import(`${pkg.name}/sri-history.json`, {
```

Comment:
Is there a reason you're doing all this manually instead of using the npm script `sri-validate`?

Reply:
Oh, and that old script is only called from Circle CI now. So once we remove the circle ci config it will get removed as well.